templates/ingress/secret.yaml

```diff
@@ -1,15 +1,15 @@
 {{- if eq (include "harbor.autoGenCertForIngress" .) "true" }}
-{{- $ca := genCA "harbor-ca" 365 }}
-{{- $cert := genSignedCert .Values.expose.ingress.hosts.core nil (list .Values.expose.ingress.hosts.core .Values.expose.ingress.hosts.notary) 365 $ca }}
+  {{- $secret := include "harbor.ingress" . }}
+  {{- $ca := genCA "harbor-ca" 365 }}
+  {{- $cert := genSignedCert .Values.expose.ingress.hosts.core nil (list .Values.expose.ingress.hosts.core .Values.expose.ingress.hosts.notary) 365 $ca }}
+  {{- $certCtx := (merge (dict "ca" $ca "cert" $cert "secret" $secret "rotateCert" .Values.expose.tls.auto.rotateCert ) .) }}
```

Comment:
Is the change of indentation in this block correct?

Reply:
Yes. Indentation doesn't really matter if you're using the `{{-` or `-}}` markers to remove whitespaces before and after the block.